### ISSUE TITLE
pretty much everything is flaky

### ISSUE THREAD

**Issue by KhafraDev** [nodejs/undici#3787]
https://github.com/nodejs/undici/actions/runs/11581098341/

6 runs failed despite none of the fails being related to the changes in the PR

- https://github.com/nodejs/undici/actions/runs/11581098341/job/32244649295#step:7:2342 (still... #3731)
- https://github.com/nodejs/undici/actions/runs/11581098341/job/32244649573#step:8:1709 (dns interceptor, @metcoder95)
- https://github.com/nodejs/undici/actions/runs/11581098341/job/32244650191#step:8:5278 (@Uzlopak you edited this last, fails on node v23)

**Comment by Uzlopak** [comment]
--no-experimental-fetch is not allowed in node23. if i make that conditional, than the diagnostic channel will be set up twice. once by native fetch and once by undici. unresolvable

**Comment by KhafraDev** [comment]
So those tests should be skipped in node v23?

**Comment by metcoder95** [comment]
Yeah, Windows has been bringing troubles here and there; I'll check them out down the week

**Comment by trivikr** [comment]
There are new flaky tests in undici in 2026, but this issue can be closed since we don't test on node 23 anymore.
